course/models.py

```diff
@@ -647,6 +647,12 @@ class CourseInstance(UrlMixin, models.Model):
         default="",
     )
 
+    sis_enroll = models.BooleanField(
+        verbose_name=_('LABEL_SIS_ENROLL'),
+        default="False",
```

Comment:
Why string `"False"`?
```suggestion
        default=False,
```